modules/parsers/parser-hdbdd/src/main/java/com/sap/xsk/parser/hdbdd/custom/ArtifactDefinitionListener.java

```diff
@@ -721,6 +720,16 @@ private void validateTopLevelSymbol(Symbol symbol) {
     }
   }
 
+  private void setContextChildSchemas(ContextSymbol symbol, String schema) {
+    symbol.getSymbols().forEach((k, v) -> {
+      if (v instanceof ContextSymbol) {
```

Comment:
Is there a need for double-checking the type?

Reply:
v is one of the symbols of a context, it could also be an entity or a view in which case we just need to set the schema.

Reply:
But why is there a need to check it again in this method? There is already an `instanceof` check where this method is called

Reply:
This is done for each of the context symbol's children, not the parameter `symbol` itself.